src/Editor/EditorView.css

```diff
@@ -0,0 +1,23 @@
+.editor-view-workspace {
```

Comment:
This was not because of line endings I think. I somehow messed up the PR by doing a rebase and then pushing things without --force. I did a reset, then pulled in new changes via git merge and pushed again. Now it looks better.